ovs/v1/ovs.yaml

```diff
@@ -534,315 +554,92 @@ paths:
-      type: string
-      example: MAEU
-      maxLength: 10
-      nullable: false
-    vesselOperatorCarrierCodeListProvider:
-      description: Identifies the code list provider used for the operator and partner carriercodes.
-      type: string
-      enum: [SMDG,NMFTA]
-      example: NMFTA
-      nullable: false
-    vesselPartnerCarrierCode:
-      description: The identifier of the vessel partner for which the current message is intended. This field allows specifying multiple, comma-separated values if there is more than one vessel partner involved.
-      type: string
-      example: MSCU,HLCU
-      maxLength: 10
-      minLength: 10
-    vesselPartnerCarrierCodeListProvider:
-      description: Identifies the code list provider used for the vessel operator and partner carrier codes. If 'vesselPartnerCarrierCode' is populated, the code list provider field is to be populated as well.
-      type: string
-      enum: [SMDG, NMFTA]
-      example: NMFTA
-    dateRange:
-      description: The time period for which schedule information is sent. The duration is populated in ISO 8601 Duration format.
-      type: string
-      format: iso8601
-      example: P4W 
-    startDate:
-      description: The start date of the period for which schedule information is sent. The value is populated in ISO 8601 Date format.
-      type: string
-      format: date
-      example: '2020-04-06'
-    transportCallID:
-      description: The unique identifier for a transport call
-      type: string
-      format: uuid
-      example: 123e4567-e89b-12d3-a456-426614174000
-      nullable: false
-  headers:
-    signatureHeader:
+      required: false
       schema:
-        type: object
-        properties:
-          keyId:
-            type: string
-            format: uuid
-            example: 123e4567-e89b-12d3-a456-426614174000
-          created:
-            type: integer
-            example: 1591960384 
-          headers:
-            type: string
-            example: "(request-target) (created) date host"
-          signature:
-            type: string
-            example: aGkgZnJvbSBkZW5tYXJr
-  parameters:    
-    limitParam:
-      name: limit
+        $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/schemas/carrierVoyageNumber'
+    UNLocationCode:  
       in: query
-      description: Maximum number of items to return.
+      name: UNLocationCode
+      description: The UN location code identifies a location in the sense of a city/a town/a village, being the smaller administrative area existing as defined by the competent national authority in each country.
       required: false
       schema:
-        type: integer
-        format: int32
-        minimum: 1
-      #  maximum: 100
-        default: 100 
-      example: 100
-    cursorParam:
-      name: cursor
+        $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/schemas/UNLocationCode'
```

Comment:
Please refer to UNLocationCode defined in DCSA_DOMAIN - if it does not exist please create it as this is globally needed